tests/migrations/test_update_mlwh_with_legacy_samples.py

```diff
@@ -0,0 +1,162 @@
+import sys
+import traceback
+from migrations.helpers.mlwh_samples_update_helper import (
+    valid_datetime_string,
+    update_mlwh_with_legacy_samples,
+)
+from crawler.constants import (
+    FIELD_ROOT_SAMPLE_ID,
+    FIELD_RNA_ID,
+    FIELD_RESULT,
+    FIELD_DATE_TESTED,
+    FIELD_LAB_ID,
+    FIELD_SOURCE,
+    FIELD_PLATE_BARCODE,
+    FIELD_CREATED_AT,
+    FIELD_UPDATED_AT,
+    MLWH_TABLE_NAME,
+    MONGO_DATETIME_FORMAT,
+)
+import pytest
```

Comment:
you don't need the above line

Reply:
import pytest? are you sure, it's in all the other test files

Reply:
Dunno, I took it out and ran it and it seemed to work. It's in conftest.py so maybe that's why?

Reply:
Removed, seems to run still